### PR TITLE
Add recipe of my nqlib package.

### PR DESCRIPTION
**Pull request by knttnk**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by knttnk** [comment]
@conda-forge/staged-recipes Could you review this PR?

**Comment by knttnk** [comment]
@conda-forge-admin, please ping team

File: recipes/nqlib/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{% set name = "nqlib" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
```
Comment (ocefpaf) [inline recipes/nqlib/meta.yaml:6]:
```suggestion
  name: {{ name|lower }}
  version: {{ version }}
```

File: recipes/nqlib/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
```
Comment (ocefpaf) [inline recipes/nqlib/meta.yaml:9]:
```suggestion
  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

File: recipes/nqlib/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    - python
+    - scipy
+    - slycot
+  run:
+    - control
+    - cvxpy
+    - numpy
+    - python
+    - scipy
+    - slycot
```
Comment (ocefpaf) [inline recipes/nqlib/meta.yaml:31]:
```suggestion
    - python >=3.6
    - numpy
    - scipy
    - control
    - cvxpy
    - slycot
```

File: recipes/nqlib/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+
+test:
+  imports:
+    - nqlib
```
Comment (ocefpaf) [inline recipes/nqlib/meta.yaml:35]:
```suggestion
test:
  imports:
    - nqlib
  commands:
    - pip check
  requires:
    - pip
```

File: recipes/nqlib/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 692b237eebb497bd8a73ac81bbd735fc1acafcbc849ee4083cf77bc55a1d7d35
+
+build:
+  number: 0
```
Comment (ocefpaf) [inline recipes/nqlib/meta.yaml:13]:
```suggestion
  noarch: python
  number: 0
```

File: recipes/nqlib/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    - pip
+
+about:
+  home: "https://github.com/knttnk/NQLib"
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: "NQLib: Library to design noise shaping quantizer for discrete-valued input control."
+  doc_url: 
+  dev_url:
```
Comment (ocefpaf) [inline recipes/nqlib/meta.yaml:44]:
```suggestion
```

**Comment by knttnk** [comment]
Thank you!

**Pull request merged**